content/footer/contents.lr

```diff
@@ -0,0 +1,11 @@
+_hidden: yes
```

Comment:
Quick note on all these `_hidden` attributes.

I'm not really using Lektor for it's primary use case, which would generate a whole site tree.  I just want a single page but keep the ability to manage different sections of content independently.  The site generation works fine with the way I have the templates set up but adding `_hidden` here makes `lektor build` skip writing those sections out as files, too.  Now it just results in the single `index.html` (as desired).